### PR TITLE
Add preview link generation for pull requests

### PR DESCRIPTION
**Pull request by wesrice**

### CODE REVIEW

File: .github/workflows/static-site.yml [hunk 1]
```diff
@@ -59,8 +56,22 @@ jobs:
       - name: Build
         run: yarn build
 
-      - name: Deploy
-        run: yarn deploy:ci
-        if: github.event.pull_request.merged == true
+      - name: Deploy to preview channel
+        if: ${{ github.event_name != 'pull_request' }}
```
Comment (kevcenteno) [inline .github/workflows/static-site.yml:60]: Why this is a `!=` instead of a `==`?

**Review by kevcenteno — COMMENTED** [review]
If a commit is added to a PR, the preview channel updated?  What happens with force pushes?

File: .github/workflows/static-site.yml (hunk 1 above)
Comment (wesrice) [reply .github/workflows/static-site.yml]: Good catch. I was testing the production step and reversed the operators of the last two steps. I just forgot to change this one back.

I've amended a commit for this and force-pushed the change.

**Comment by wesrice** [comment]
@kevcenteno From what I've seen, the comment is updated with a reference to the latest commit hash, and the new assets are deployed to the same preview link.

**Pull request merged**